roles/vendors/kvm/tasks/iso.yml

```diff
@@ -163,7 +163,8 @@
     method: POST
     body_format: json
     body: { "ResetType": "ForceOn" }
-    status_code: [200, 204]
+    # redfish specification: https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.12.1.pdf
```

Comment:
Looks like its been bumped and that link is no longer valid. 
https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.19.0.pdf

Reply:
Weird, I tried the link again but it was working :( anyway, I'll update the change with your link, since it's newer.